internal/usage/usage.go

```diff
@@ -403,33 +403,33 @@ dbName and collection are required only for built-in roles.`
 	ReadConcern                               = "Default level of acknowledgment requested from MongoDB for read operations set for this cluster."
 	WriteConcernAdvancedSettings              = "Default level of acknowledgment requested from MongoDB for write operations set for this cluster."
 	TLSProtocol                               = "Minimum Transport Layer Security (TLS) version that the cluster accepts for incoming connections."
-	DisableTableScan                          = "Disables executing any query that requires a collection scan to return results."
-	EnableTableScan                           = "Enables executing any query that requires a collection scan to return results."
-	DisableFailIndexKeyTooLong                = "Disables writing documents that exceed 1024 bytes without indexing."
-	EnableFailIndexKeyTooLong                 = "Enables writing documents that exceed 1024 bytes without indexing."
-	DisableJavascript                         = "Disables the execution of operations that perform server-side executions of JavaScript."
-	EnableJavascript                          = "Enables the execution of operations that perform server-side executions of JavaScript."
+	DisableTableScan                          = "Flag that disables executing any query that requires a collection scan to return results."
+	EnableTableScan                           = "Flag that enables executing any query that requires a collection scan to return results."
+	DisableFailIndexKeyTooLong                = "Flag that disables writing documents that exceed 1024 bytes without indexing."
+	EnableFailIndexKeyTooLong                 = "Flag that enables writing documents that exceed 1024 bytes without indexing."
+	DisableJavascript                         = "Flag that disables the execution of operations that perform server-side executions of JavaScript."
+	EnableJavascript                          = "Flag that enables the execution of operations that perform server-side executions of JavaScript."
 	OplogMinRetentionHours                    = "Minimum retention window for cluster's oplog expressed in hours."
 	OplogSizeMB                               = "Storage limit of cluster's oplog expressed in megabytes."
 	SampleRefreshIntervalBIConnector          = "Interval in seconds at which the mongosqld process re-samples data to create its relational schema."
 	SampleSizeBIConnector                     = "Number of documents per database to sample when gathering schema information."
 	ExporterClusterName                       = "One or more comma separated cluster names to import"
-	OperatorIncludeSecrets                    = "Generate kubernetes secrets with data for projects, users, deployments entities" //nolint:gosec //This is just a message, not a credential
+	OperatorIncludeSecrets                    = "Flag that generates kubernetes secrets with data for projects, users, deployments entities."
```

Comment:
I believe the comment that was removed here is required to pass linting due to the use of the word "secrets"? But not sure